### ISSUE TITLE
Unnecessary dependency on Python

### ISSUE THREAD

**Issue by me-and** [me-and/Cygwin-Git#26]
Should split of the p4 package so the core Git package no longer depends on Python.

https://cygwin.com/ml/cygwin/2016-05/msg00195.html

**Comment by svnpenn** [comment]
Yes please. And if you really wanted to go crazy you should remove the Perl
dependency too. Perl is only needed for some of the ancillary scripts like
"git send-email". You could do it similar to "vim-minimal" or "perl_base":

~~~
@ git-minimal
requires: bash cygutils cygwin less libcurl4 libexpat1 libiconv2 libintl8
libopenssl100 libpcre1 openssh rsync zlib0

@ git
requires: bash cygutils cygwin less libcurl4 libexpat1 libiconv2 libintl8
libopenssl100 libpcre1 openssh perl-Carp perl-Error perl-TermReadKey perl_base
python python2 rsync zlib0
~~~

**Comment by svnpenn** [comment]
@me-and 

http://cygwin.com/ml/cygwin/2017-06/msg00162.html

**Comment by me-and** [comment]
Fixed with fce6e477
